compiler/rustc_mir_transform/src/check_unsafety.rs

```diff
@@ -614,6 +619,32 @@ pub fn check_unsafety(tcx: TyCtxt<'_>, def_id: LocalDefId) {
     }
 }
 
+fn suggest_wrapping_unsafe_block(
+    tcx: TyCtxt<'_>,
+    def_id: LocalDefId,
+    err: &mut DiagnosticBuilder<'_, ()>,
+) {
+    let body_span = tcx
+        .hir()
+        .body(tcx.hir().body_owned_by(tcx.hir().local_def_id_to_hir_id(def_id)))
+        .value
+        .span;
+
+    let suggestion = vec![
+        (
+            tcx.sess.source_map().start_point(body_span).shrink_to_hi(),
+            " /* FIXME */ unsafe {".into(),
```

Comment:
Why the "FIXME"? Do we have other suggestions that do something like this?

Reply:
No, I don't think so. I just put it there because I thought the idea was interesting, which tells you how bad I am at diagnostics. Just something I can easily remove anyway

Reply:
I don't think I quite see the need for such a comment. But if there is a comment, it should at least explain what needs fixing here. A bare FIXME is fairly useless.

Reply:
I don't think we should insert such a comment. A whole-function `unsafe` block will already be something people may wish to fix, and people may also use `git diff` to review and edit applied suggestions.